jellyfish/_jellyfish.py

```diff
@@ -218,6 +217,79 @@ def hamming_distance(s1, s2):
 
     return distance
 
+def wagner_fischer_distance(s1, s2, trace = False):
+    _check_type(s1)
+    _check_type(s2)
+
+    traced = ''
+    if len(s1) == 0 and len(s2) == 0:
+        if trace == False:
+            return 0
+        else:
+            return 0, traced
+    if len(s1) == 0:
+        if trace == False:
+            return len(s2)
+        else:
+            for each in s2:
+                traced = traced + '~'
+            return len(s2), traced
+    if len(s2) == 0:
+        if trace == False:
+            return len(s1)
+        else:
+            for each in s1:
+                traced = traced + '~'
+            return len(s1), traced
+    # the rows must be smaller than the column, will help with tracing.
+    if len(s1) > len(s2):
+        temp = s1
+        s1 = s2
+        s2 = temp
+    # dynamic programming algorithm to compute edit distance between two strings.
+    # https://en.wikipedia.org/wiki/Wagner%E2%80%93Fischer_algorithm
+    n = len(s1) + 1 #rows
+    m = len(s2) + 1 #columns
+    # defining a matrix of n X m
+    d = [[0 for x in range(m)] for y in range(n)]
+    # initialization
+    for i in range(n):
+        d[i][0] = i
+    for j in range(m):
+        d[0][j] = j
+    # filling up the rest of matrix by Flood Filling
+    for j in range(1, m):
+        for i in range(1, n):
+            if s1[i-1] == s2[j-1]:
+                d[i][j] = d[i-1][j-1]
+            else:
+                d[i][j] = min(d[i-1][j] + 1, \
+                              d[i][j-1] + 1, \
+                              d[i-1][j-1] + 1 )
+    if trace == False:
```

Comment:
Maybe reverse this in favor of the argument ``None``. (And PEP8 doesn't like it) 

```
if trace:
    # code
else:
    return d[n-1][m-1]
```